librina/include/librina/ipc-daemons.h

```diff
@@ -198,10 +198,28 @@ class FwdCDAPMsgEvent: public IPCEvent {
 	 * in the direction IPC Process --> IPC Manager. */
 	int result;
 
-	FwdCDAPMsgEvent(const ser_obj_t& sm, int result,
+	FwdCDAPMsgRequestEvent(const ser_obj_t& sm, int result,
 			unsigned int sequenceNumber);
 };
 
+/**
+ * The IPC Manager wants to forward a CDAP response message to
+ * an IPC process, or the IPC process wants to forwards a
+ * CDAP response back to the IPC Manager.
+ */
+class FwdCDAPMsgResponseEvent: public IPCEvent {
```

Comment:
This class FwdCDAPMsgResponseEvent is just a clone of FwdCDAPMsgRequestEvent. Do we need it?